### PR TITLE
fix: additional vertex collection operations

### PR DESCRIPTION
**Pull request by sergeda**
Also fixes some formatting issue

### CODE REVIEW

File: arangodb-http/src/it/scala/io/funkode/arangodb/http/ArangoJsonIT.scala [hunk 1]
```diff
@@ -67,9 +67,20 @@ trait ArangoExamples:
   val secondCountries = List(Country("🇦🇫", "Afghanistan"), Country("🇦🇬", "Antigua and Barbuda"))
 
   val politics = GraphName("politics")
+  val geography = GraphName("geography")
   val allies = CollectionName("allies")
+  val geographyCollection = CollectionName("geography")
+  val geographyCollection2 = CollectionName("geography2")
```
Comment (carlos-verdes) [inline arangodb-http/src/it/scala/io/funkode/arangodb/http/ArangoJsonIT.scala:73]: I think this is not used on this file, maybe you wanted to use on line 81?

**Review by carlos-verdes — COMMENTED** [review]
Small comment on the test, take a look

File: arangodb-http/src/it/scala/io/funkode/arangodb/http/ArangoJsonIT.scala (hunk 1 above)
Comment (sergeda) [reply arangodb-http/src/it/scala/io/funkode/arangodb/http/ArangoJsonIT.scala:73]: Nice catch. Updated the code

**Review by carlos-verdes — APPROVED** [review]
LGTM and merging

**Pull request merged**